royal-society/_journals.tab

```diff
@@ -6,3 +6,6 @@ Open Biology		2046-2441	http://rsob.royalsocietypublishing.org/site/misc/stylean
 Philosophical Transactions of the Royal Society A	1364-503X	1471-2962	http://rsta.royalsocietypublishing.org/site/misc/style-guide.xhtml#question9	physics
 Philosophical Transactions of the Royal Society B	0962-8436	1471-2970	http://rstb.royalsocietypublishing.org/	biology
 Proceedings of the Royal Society A	1364-5021	1471-2946	http://rspa.royalsocietypublishing.org/site/misc/preparing-your-article.xhtml#question3	physics
+Royal Society Open Science	2054-5703	x	https://royalsociety.org/journals/authors/author-guidelines/	science
+Proceedings of the Royal Society B	1364-5021	1471-2946	https://royalsociety.org/journals/authors/author-guidelines/	biology
+Notes and Records: the Royal Society Journal of the History of Science	0035-9149	1743-0178	https://royalsociety.org/journals/authors/author-guidelines/	history
```

Comment:
@damnation333, "Proceedings of the Royal Society B" was absent because that's the independent parent in this case (see https://github.com/citation-style-language/styles/blob/master/proceedings-of-the-royal-society-b.csl, and the style template file). We could add it here, but then we would also have to put it on the skip list.

Also, are you sure "Notes and Records" uses the same format as the other journals? Compare e.g. http://rsnr.royalsocietypublishing.org/content/roynotesrec/70/4/305.full.pdf and http://rsnr.royalsocietypublishing.org/author-information#question2 to http://rsos.royalsocietypublishing.org/content/royopensci/4/1/160645.full.pdf and http://rsos.royalsocietypublishing.org/author-information#question5.